server/gabble/chat/health_events.go

```diff
@@ -2,7 +2,6 @@ package chat
 
```

Comment:
Because gabble is more of a package than an implementation of a chat service.
As you can see in the *main.go*, gabble instance created instead of calling a set of functions in order to start our server.

Semantically talking `gabble` is a package with a `chat` as a component of this package.

Reply:
```go
package main

import (
	"github.com/whizzes/gabble/server/gabble"
	"github.com/whizzes/gabble/server/gabble/config"
)

// We are importing from `gabble` the components required to create
// a gabble chat instance.
// gabble beign a package with all of its components inside
// if you want to create an instance of `gabble` in another
// project gabble is package ready/module.
func main() {
	conf, err := config.FromEnv()

	if err != nil {
		panic(err)
	}

	server, err := gabble.NewGabble(conf)

	if err != nil {
		panic(err)
	}

	server.Run()
}
```